plugin.audio.calmradio/addon.py

```diff
@@ -347,4 +350,4 @@ def update_artwork(channel, recent_tracks_url):
         executebuiltin('Notification("{0}", "{1}", 6000, "special://home/addons/{2}/icon.png")'
                        .format(ADDON.getLocalizedString(30000),
                                ADDON.getLocalizedString(32202),
-                               ADDON_ID))
+                               ADDON_ID))
```

Comment:
you´re removing the line breaks at end of file :) make sure to set up your editor correctly.